parts/linux/cloud-init/artifacts/components.json

```diff
@@ -443,6 +443,11 @@
             ]
           }
         },
+        "marinerkata": {
```

Comment:
could we just have logic directly in the install scripts to skip containerd installation if kata is enabled? adding a new object here just for an empty array seems unnecessary

I guess we can leave it like this if we think that we'll have divergent versions of containerd needed for kata-enabled SKUs in the future

Reply:
That's a good call out indeed. 
We want the logic can be completely visible in the centralized component.json. If we put the logic in the scripts to skip containerd installation for Kata build, then someone will need to look into the scripts again to figure out why kata build doesn't install containerd. And you also made a good point that, in the future Kata build could be requesting divergent versions for MarinerKata v2, v3, etc. So I would tend to keep it like this.

Reply:
what do you think about adding a new keyword to the components schema, something like `base` as a valid version string that indicates we should just take it from the base image?

I think something like that might be a bit more readable/understandable than this

Reply:
Good idea! Let me try <BASE>

Reply:
I ended up using `<SKIP>` because in this case it's not using the containerd from base image. There is another step installing a custom containerd for kata so SKIP should be more descriptive.